Tests/Realm.Tests/Database/ObjectSchemaTests.cs

```diff
@@ -25,6 +25,8 @@
 
 namespace Realms.Tests.Database
 {
+#pragma warning disable CS0618 // ObjectSchema.Builder(string, bool) is obsolete, Builder(string, ObjectSchemaType) should be used instead
```

Comment:
Do we need to disable this for the entire file? Can we migrate all the tests to use the new constructor and if you want to test the old one, then create a new test and mark it `[Obsolete]`?